runtime/strms_sess.c

```diff
@@ -288,7 +288,7 @@ BEGINObjClassInit(strms_sess, 1, OBJ_IS_CORE_MODULE) /* class, version - CHANGE
 	CHKiRet(objUse(prop, CORE_COMPONENT));
 
 	CHKiRet(objUse(glbl, CORE_COMPONENT));
-	iMaxLine = glbl.GetMaxLine(); /* get maximum size we currently support */
+	iMaxLine = 8096; /* get maximum size we currently support */
```

Comment:
This does not look right, given the previous code. Is it that at this stage we do not have the config object fully populated? In any case, if I am not mistaken, we should have one. So better put the constant in it's initialization. Or am I overlooking something (might very well be, haven't looked at this code area for quite a while).

Reply:
Yes, you are right. I remember that I set it to a constant value when I was debugging something and then forgot about it. I am not familiar with this part of code, is it even used? I could not find any references where the ```strms_sess```  initializer is even invoked(although I might overlooked something). The last call I could found has been removed via commit https://github.com/rsyslog/rsyslog/commit/ae8d9853be406a2b5dd8b7353ab0fc6f99e41b3c.

Reply:
let me dig into it, will take a short while

Reply:
I overlooked it's the class initializer (yeah, you said it... ;-). In this case: yes it is called, it's done by the module loader.

But: I don't see that `iMaxline` is actually being used! I'll craft a small PR to remove it and see if something really strange comes up. Should not be the case as it is a static variable and nowhere else referenced.

Reply:
Indeed, `iMaxLine` was no longer used. It's gone now. If you rebase to master, you have one problem less ;-)

Reply:
I updated the branch with the commits from master and also applied changes according to your comments :)